ams/server/src/main/java/com/netease/arctic/server/utils/IcebergTableUtil.java

```diff
@@ -78,9 +80,10 @@ public static Snapshot getSnapshot(Table table, boolean refresh) {
     return table.currentSnapshot();
   }
 
-  public static Optional<Snapshot> findSnapshot(Table table, Predicate<Snapshot> predicate) {
-    Iterable<Snapshot> snapshots = table.snapshots();
-    return Iterables.tryFind(snapshots, predicate);
+  public static Optional<Snapshot> findSnapshotDesc(Table table, Predicate<Snapshot> predicate) {
```

Comment:
I am not sure `findSnapshotDesc` is a good enough name for this method.
IMO, this method seems to try to find a snapshot matching a specific condition, so maybe `findFirstMatchSnapshot` is a proper name?